lib/resty/websocket/client.lua

```diff
@@ -128,6 +128,13 @@ function _M.connect(self, uri, opts)
             end
             ssl_verify = true
         end
+
+        if opts.headers then
+          headers = opts.headers
```

Comment:
I think we use 4-space indentation consistently in this library. Please fix other similar places.

Reply:
Fixed.